LFS-365: When a user needs to relogin to complete an action such as saving a form, show the login dialog inline

veronikaslc opened the pull request.

Commit:
LFS-365: When a user needs to relogin to complete an action such as saving a form, show the login dialog inline

Commit:
[misc] Code cleanup

sdumitriu commented on modules/data-entry/src/main/frontend/src/questionnaire/Form.jsx:
```diff
@@ -121,14 +123,17 @@ function Form (props) {
+    setLoginDialogShow(false);
+  }
+
+  let handleLoginClose= (success) => {
```
Where is this used?

sdumitriu reviewed (changes requested):
This doesn't behave properly:
- entering wrong credentials closes the dialog, the error is displayed for a split second only
- entering correct credentials works, but the save request seems to be triggered before the login completes, so saving still fails

Commit:
LFS-365: When a user needs to relogin to complete an action such as saving a form, show the login dialog inline
Code cleanup

Commit:
LFS-365: When a user needs to relogin to complete an action such as saving a form, show the login dialog inline
Let the login dialog stay if login failed and show error; Hide dialog if
login is successfull.

veronikaslc commented:
@sdumitriu Codereview comments addressed
